user-guide/Advanced_Functionality/Security/DataMiner_hardening_guide.md

```diff
@@ -53,6 +53,13 @@ For the inter-DMA communication, like for the communication with DataMiner Cube,
 
 To enable gRPC for the communication between DataMiner Agents in a cluster, add [redirects in DMS.xml](xref:DMS_xml#redirects-subtag) or, from **10.3.6/10.3.0 [CU3] onwards**, disable .NET Remoting completely in [MaintenanceSettings.xml](xref:Configuration_of_DataMiner_processes#disabling-net-remoting).
 
+#### NATS
+
+By default, NATS does not employ TLS encryption, leaving communication susceptible to eavesdropping. Consequently, we strongly recommend [enabling TLS encryption for enhanced security within your NATS cluster](xref:Security_NATS).
+
+> [!IMPORTANT]
+> TLS encryption over NATS is exclusively accessible starting from DataMiner 10.4.3 CU0 onwards when employing CA-Signed Certificates (Certificate Authority-Signed Certificates). It's crucial to be aware that the utilization of Self-Signed Certificates may result in DataMiner startup failures. 
```

Comment:
The release note that adds this feature is [RN38302](https://intranet.skyline.be/DataMiner/Lists/Release%20Notes/DispForm2.aspx?ID=38302).

To clarify, the 10.4.3 version, which is now sealed is working with trusted CA-Signed Certificates, while the 10.4.4 version will work with self-signed certificates. Please note that the 10.4.3 would in theory already work with self-signed certificates, which was unfortunately not possible due to development issues.
Therefore the procedure for the self-signed was already created in the Secure NATS section, which can be adjusted in order to indicate that or it can directly indicated in this page.

Reply:
@FlavioJA, when I check the description of RN 38302, this seems to apply to the BrokerGateway soft-launch feature only: "*The MessageBroker now supports TLS (Transport Layer Security) configuration, enhancing the security of communication between NATS clients and the NATS server. This feature is available when using BrokerGateway*". 
Is this indeed only available if the *BrokerGateway* soft-launch option is enabled? Or is this actually a general change, despite what is suggested in the RN?
If it is a soft-launch feature, I'm not sure we should add this PR in its current form yet, because we only allow the use of soft-launch features on staging platforms, but users reading the new page will assume they need to apply this on their production platforms as well.

Reply:
The BrokerGateway itself is a soft-launch option. However, it is not really needed to achieve TLS over NATS if the nats-server.config is configured manually, as we describe in the Secure NATS section.